iModelCore/ECDb/ECDb/SchemaManagerDispatcher.cpp

```diff
@@ -1075,6 +1075,72 @@ DropSchemaResult MainSchemaManager::DropSchema(Utf8StringCR name, SchemaImportTo
     return rc;
 }
 
+/*---------------------------------------------------------------------------------------
+* @bsimethod
++---------------+---------------+---------------+---------------+---------------+------*/
+DropSchemaResult MainSchemaManager::DropSchemas(T_Utf8StringVectorCR schemaNames, SchemaImportToken const* schemaImportToken, bool logIssue) const

```

Comment:
You need to expose this via `DgnDb` as that is the one that hold `scheamImportToken`. When you merge this back in imodel02 the client code will call the method on `DgnDb::DropSchemas()`. Follow `DropSchema()` and add `DropSchemas()` version.

Reply:
Done